File: streaming-server/src/main/java/io/github/hs96wings/streaming_server/chat/dto/MyChatListResDto.java
package io.github.hs96wings.streaming_server.chat.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class MyChatListResDto {
    private Long roomId;
    private String roomName;
    private String isGroupChat;
    private Long unReadCount;
}

File: streaming-server/src/main/java/io/github/hs96wings/streaming_server/chat/dto/MyChatListResDto.kt
package io.github.hs96wings.streaming_server.chat.dto

import lombok.AllArgsConstructor
import lombok.Builder
import lombok.Data
import lombok.NoArgsConstructor

data class MyChatListResDto @JvmOverloads constructor(
    val roomId: Long,
    val roomName: String,
    val isGroupChat: Boolean,
    val unReadCount: Long
)
